routing/blindedpath/blinded_path_test.go

```diff
@@ -547,35 +547,59 @@ func genBlindedRouteData(rand *rand.Rand) *record.BlindedRouteData {
 // an example mentioned in this spec document:
 // https://github.com/lightning/bolts/blob/master/proposals/route-blinding.md
 // This example does not use any dummy hops.
+// Added an additional node (Dave), increasing the possible paths, which can
+// then be filtered using the MaxNumPaths (maximum blinded paths) option.
```

Comment:
I think Elle mentioned this in your other PR, I am in favour of always adding new unit-tests rather than trying to change old ones. With the new commit structure this should be easy to do. I don'T think we need to check the blinded path values in this new test but rather only check if the right amount of route is returned.